statler/urls.py

```diff
@@ -15,13 +15,11 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
-#import statler_api
-#import statler_site
 
 urlpatterns = [
-    url(r'^api/', include('statler_api.urls')),
+    url(r'^api/', include('statler_api.urls', namespace="api")),
```

Comment:
Awesome. We should be able to use the namespace + url name to dynamically fill the target of the site pages' ajax requests, correct? (This is probably just something to keep in mind for future work)